### PR TITLE
Axis documentation

### PR DESCRIPTION
**Pull request by Kane610**
**Description:**
Documentation for Axis platform

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant):** home-assistant/home-assistant#7381

### CODE REVIEW

**Review by Landrash — COMMENTED** [review]
Some minor changes I would suggest.

File: source/_components/axis.markdown [hunk 1]
```diff
@@ -0,0 +1,74 @@
+---
+layout: page
+title: "Axis"
```
Comment (Kane610) [inline source/_components/axis.markdown]: Looking on other hubs they're not using 'hub' in the title.

File: source/_components/axis.markdown [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+sharing: true
+footer: true
+logo: axis.png
+ha_category: Hub
+ha_release: "0.44"
+---
+
+[Axis Communications](https://www.axis.com/) devices are surveillance cameras and other security related network connected hardware. Sensor API works with firmware 5.50 and newer.
+
+You will be prompted to configure your devices through the Home Assistant interface, enter the security key when prompted and click configure. You can later change configuration in your Home Assistant directory
```
Comment (Kane610) [inline source/_components/axis.markdown]: Sure

Comment (Kane610) [inline source/_components/axis.markdown]: I shouldn't mention configuring through gui?

File: source/_components/axis.markdown [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+You can also add your devices manually to your `configuration.yaml` file:
```
Comment (Kane610) [inline source/_components/axis.markdown]: Fixed

File: source/_components/axis.markdown [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+Configuration variables:
+
+- **device** (*Required*): Unique name 
+- **host** (*Required*): The IP address to your Axis device.
+- **username** (*Optional*): The username to your Axis device. Default 'root'.
+- **password** (*Optional*): The password to your Axis device. Default 'pass'.
+- **scan_interval** (*Optional*): How long (in full seconds) a sensors value should keep its positive value. Default 0.
+- **location** (*Optional*): Physical location of your Axis device. Default not set.
+- **include** (*Required*): This cannot be empty else there would be no use adding the device at all.
```
Comment (Kane610) [inline source/_components/axis.markdown]: Separated them with a line break

File: source/_components/axis.markdown (hunk 2 above)
Comment (Landrash) [inline source/_components/axis.markdown]: Was more referring to this part
>You will be prompted to configure your devices through the Home Assistant interface

It's up to you but often less is more.

**Review by Landrash — APPROVED** [review]

**Comment by Landrash** [comment]
Looks good and can be merged when parent PR is merged.

**Review by balloob — APPROVED** [review]
Ok to merge when versions are updated.

**Pull request merged**